tests/test_badpixel.py

```diff
@@ -0,0 +1,91 @@
+import os
+import pytest
+import corgidrp
+import numpy as np
+import corgidrp.mocks as mocks
+from corgidrp.mocks import create_default_headers
+from corgidrp.l2a_to_l2b import correct_bad_pixels
+from corgidrp.data import Image, Dataset, BadPixelMap
+
+old_err_tracking = corgidrp.track_individual_errors
+
+data = np.ones([1024,1024])*2.
+err = np.ones([1024,1024]) *0.5
+dq = np.zeros([1024,1024], dtype = np.uint16)
+prhd, exthd = create_default_headers()
+
+def test_bad_pixels():
+
+    corgidrp.track_individual_errors = True
```

Comment:
tracking individual errors is a best effort mode with the default mode being to not track the errors. I think we probably want to set this to False for testing